From d1a2c998a00caa17128e9b5afbf3809c94574aa7 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <philippe.gerum@exail.com>
Date: Sun, 8 Jun 2025 18:20:12 +0200
Subject: [PATCH 0942/1013] evl/net: leave the experimental state

The out-of-band network stack is no more experimental, drop the
related caveat notice from the Kconfig help text.

Signed-off-by: Philippe Gerum <philippe.gerum@exail.com>
---
 kernel/evl/Kconfig | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/kernel/evl/Kconfig b/kernel/evl/Kconfig
index 83e5c97c21f2..604a1b8f4b2e 100644
--- a/kernel/evl/Kconfig
+++ b/kernel/evl/Kconfig
@@ -89,7 +89,7 @@ config EVL_RUNSTATS
 	the /sys interface.
 
 config EVL_NET
-        bool "Out-of-band networking (EXPERIMENTAL)"
+        bool "Out-of-band networking"
 	default n
 	select NET_OOB
 	select NET_SCHED
@@ -97,11 +97,11 @@ config EVL_NET
 	select INET
 	select VLAN_8021Q
 	help
-	This option enables preliminary networking support for EVL.
+	This option enables out-of-band networking support for EVL.
 
-	CAUTION! This is WIP, experimental code with limited support
-	at the moment, which is still subject to significant UAPI and
-	kernel API changes all over the map.
+	The out-of-band network stack provides datagram-oriented
+	networking support to applications with real-time
+	communication requirements (UDP and raw packet protocols).
 
 menu "Fixed sizes and limits"
 
-- 
2.30.2

